### PR TITLE
Adds new profession: Kitted Survivor

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Added new Profession 'Kitted Survivor'"

#### Describe the solution

Allows players to start with a survival kit, which might aid them.

#### Describe alternatives you've considered

Not adding this profession.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -4026,6 +4026,18 @@
       "female": [ "bra", "boy_shorts" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "kit_survivor",
+    "name": "Kitted Survivor",
+    "description": "At the local mall, you saw a sign advertising a discount on survival kits. You bought one, more for show than for actual use. Now, it's all you have."
```
Comment (anothersimulacrum) [inline data/json/professions.json]:
```suggestion
    "description": "At the local mall, you saw a sign advertising a discount on survival kits. You bought one, more for show than for actual use. Now, it's all you have.",
```

**Comment by AMurkin** [comment]
And fix the summary.
SUMMARY: Content "Added new Profession 'Kitted Survivor'"

File: data/json/professions.json (hunk 1 above)
Comment (Xenomorph-III) [reply data/json/professions.json]: Oh - how do I lint my JSON?

File: data/json/professions.json [hunk 2]
```diff
@@ -4026,6 +4026,18 @@
       "female": [ "bra", "boy_shorts" ]
     }
   },
+  {
+    "type": "profession",
+    "ident": "kit_survivor",
+    "name": "Kitted Survivor",
+    "description": "At the local mall, you saw a sign advertising a discount on survival kits. You bought one, more for show than for actual use. Now, it's all you have.",
```
Comment (AMurkin) [inline data/json/professions.json]:
```suggestion
    "description": "At the local mall, you saw a sign advertising a discount on survival kits.  You bought one, more for show than for actual use.  Now, it's all you have.",
```

File: data/json/professions.json (hunk 1 above)
Comment (AMurkin) [reply data/json/professions.json]: Lint the whole file contents with a http://dev.narc.ro/cataclysm/format.html

File: data/json/professions.json [hunk 3]
```diff
@@ -4026,6 +4026,18 @@
...
+    "type": "profession",
+    "ident": "kit_survivor",
+    "name": "Kitted Survivor",
+    "description": "At the local mall, you saw a sign advertising a discount on survival kits.  You bought one, more for show than for actual use.  Now, it's all you have.",
+    "points": 1,
+    "items": {
+      "both": [ "sneakers", "socks", "jeans", "tshirt", "wristwatch", "jacket_light", "survival_kit" ],
+      "male": [ "boxer_shorts" ],
+      "female": [ "panties", "bra" ]
+    },
```
Comment (AMurkin) [inline data/json/professions.json]:
```suggestion
    }
```

**Pull request merged**